lib/KeyboardAwareScrollView.js

```diff
@@ -20,6 +20,14 @@ const KeyboardAwareScrollView = React.createClass({
     this.setResetScrollToCoords(this.props.resetScrollToCoords)
   },
 
+  scrollTo: function (...args) {
+    return this.refs._rnkasv_keyboardView.scrollTo(...args)
+  },
+
+  scrollToEnd: function (...args) {
```

Comment:
`scrollToEnd` and `scrollWithoutAnimationTo` should live in the mixin, because the `ListView` also has these functions and implementing them in the `ScrollView` only would avoid users to have that in the `ListView`.